### PR TITLE
Add content/pt-br/cloud_native_tech.md

### PR DESCRIPTION
**Pull request by MrErlison**
Changes
* Add cloud_native_tech.md

### CODE REVIEW

File: content/pt-br/cloud_native_tech.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Tecnologia Nativa da Nuvem
+status: Completed
+category: Conceito
+---
+
+## O que é
+
+As tecnologias nativas da nuvem, também conhecidas como a pilha nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```
Comment (jihoon-seo) [inline content/pt-br/cloud_native_tech.md:9]: [The `pt-br` version of the term `Cloud Native Apps`](https://deploy-preview-683--cncfglossary.netlify.app/pt-br/cloud_native_apps/) exists, so updated the link. 😊

```suggestion
As tecnologias nativas da nuvem, também conhecidas como a pilha nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```

**Review by jihoon-seo — COMMENTED** [review]
Preview: https://deploy-preview-683--cncfglossary.netlify.app/pt-br/cloud_native_tech/

@MrErlison Thank you for your contribution. 😊
And PTAL!

File: content/pt-br/cloud_native_tech.md (hunk 1 above)
Comment (MrErlison) [reply content/pt-br/cloud_native_tech.md]: Sorry about that. I will keep an eye out.

File: content/pt-br/cloud_native_tech.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Tecnologia Nativa da Nuvem
+status: Completed
+category: Conceito
```
Comment (edsoncelio) [inline content/pt-br/cloud_native_tech.md:4]: Conceito to conceito

File: content/pt-br/cloud_native_tech.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+## O que é
+
+As tecnologias nativas da nuvem, também conhecidas como a pilha nativa da nuvem, são as tecnologias usadas para criar [aplicativos nativos da nuvem](/pt-br/cloud_native_apps/). Essas tecnologias permitem que as organizações criem e executem aplicações escaláveis em ambientes modernos e dinâmicos, tais como as nuvens públicas, privadas e  híbridas, ao mesmo tempo em que aproveitam ao máximo os benefícios da [computação em nuvem](/cloud_computing/). Eles são projetados desde o início para explorar os recursos de computação em nuvem e contêineres, serviços em malha, microsserviços e infraestrutura imutável exemplificam essa abordagem.
```
Comment (edsoncelio) [inline content/pt-br/cloud_native_tech.md:9]: `Pilha` sounds out of context here, maybe we can use the original term? (*stack*)

**Review by edsoncelio — CHANGES_REQUESTED** [review]

File: content/pt-br/cloud_native_tech.md (hunk 3 above)
Comment (MrErlison) [reply content/pt-br/cloud_native_tech.md:9]: I don't like pilha but mix stack with portuguese is strange for me. But you all right. Sometimes I like see the RedHat's translation and stack in portuguese is stack.

**Review by edsoncelio — APPROVED** [review]

**Pull request merged**